app/services/foreman_templates/template_exporter.rb

```diff
@@ -77,8 +77,8 @@ def get_template_filename(template)
 
     def get_dump_dir(template)
       kind = template.respond_to?(:template_kind) ? template.template_kind.try(:name) || 'snippet' : nil
-      template_class_dir = template.model_name.human.pluralize.downcase.tr(' ', '_')
-      template_class_dir += '_templates' if template_class_dir == 'partition_tables'
+      template_class_dir = template.model_name.plural.tr(' ', '_')
```

Comment:
I think you do not need the `.tr(' ', '_')` part as pluralized model names do not have spaces

Reply:
oh sure, fixed :-)